app.py

```diff
@@ -8,6 +8,13 @@
 from flask_assets import Environment
 from flask_babel import Babel, gettext, pgettext
 from flask_compress import Compress
+
+try:
+    from flask_debugtoolbar import DebugToolbarExtension
+
+    toolbar = DebugToolbarExtension()
```

Comment:
I wonder if we should also only set it if the flask env isn't production - our dependency management is much more consistent now and I'm not sure what the failure mode is but that would also be consistent with a few other gated checks